astropy/__init__.py

```diff
@@ -11,7 +11,8 @@
 import os
 from warnings import warn
 
-__minimum_numpy_version__ = '1.10.0'
+__minimum_python_version__ = '3.5'
+__minimum_numpy_version__ = '1.13.0'
```

Comment:
`__minimum_python_version__` should not appear here ? (maybe a rebase conflict ?)

Reply:
Thanks - I think I intended to move it up here, but seem to have forgotten to delete the definition lower down.